File: python/stdlib/string/concat.py
try:
    from ...stdlib import Node as base
except ValueError:
    from stdlib import Node as base


class Node(base.Node):
    def __init__(self, verbose, args):
        super(Node, self).__init__("String Concat", "string.concat", "", {"left": "String", "right": "String"},
                                   {"result": "String"},
                                   "Concat the two strings.", verbose)

    def tick(self, value):
        return {"result": value["left"] + value["right"]}

File: lua/stdlib/string/concat.lua
local myNode = {}

function myNode.init(node)
  node.tick = function(value)
    result = {}
    result.result = value.left .. value.right
    return result
  end
end

function myNode.spec(node)
  node.name = "String Concat"
  node.inputs.left = "String"
  node.inputs.right = "String"
  node.outputs.result = "String"
  node.desc = "Concat left and right."
end

return myNode
